packages/manager/modules/cda/src/ip/delete/cda-ip-delete.html

```diff
@@ -5,7 +5,7 @@
 >
     <oui-modal
         heading="{{ ::'cda_ip_delete_title' | translate }}"
-        primary-label="{{ ::'delete' | translate }}"
```

Comment:
Following translation key is not available into the `ovhManagerCdaIp` module.
Here as a hotfix, I'm using a translation available from the same given module.

Maybe we can consider to rename this translation key in order to avoid using some `common` translations.
But this can be done in a separate Pull Request if needed.